assembly/integer/u256.ts

```diff
@@ -202,6 +227,79 @@ export class u256 {
     return new u256(a.lo1 & b.lo1, a.lo2 & b.lo2, a.hi1 & b.hi1, a.hi2 & b.hi2);
   }
 
+  @inline @operator('>>')
+  static shr(value: u256, shift: i32): u256 {
+      shift &= 255;
+
+      // need for preventing redundant i32 -> u64 extends
+      var shift64 = shift as u64;
+      let lo1: u64, lo2: u64, hi1: u64, hi2: u64;
+
+      if (shift64 <= 64) {
```

Comment:
I think for better compare with 32-bit `shift` instead `shift64`

Reply:
Sure. Should I change the one in u128 implementation as well?

Reply:
yes, indeed

Reply:
It seems that when I changed the line in `u128` to `shift32 = shift as u32`, many tests fail.

Reply:
no, I mean leave `var shift64 = shift as u64;` line, but use `shift` param (which i32) during comparison, like:
```ts
if (shift <= 64) { // <- here use `shift`
  hi2 = value.hi2 >> shift64; // <- here use shift64
  ...
  lo1 = (value.lo1 >> shift64) | (lo2 << (64 - shift64));
}
```

Reply:
I see